school_fees/models/school_fees.py

```diff
@@ -91,8 +97,8 @@ def fees_register_confirm(self):
                 ):
                     raise ValidationError(
                         _(
-                            """There is already a Payslip exist for student: %s for same date.!
-"""
+                            "There is already a Payslip exist for student: "
+                            "%s for same date.!"
```

Comment:
make it better.
'There exists a Fees record for %s student name'

Do we call it a payslip for students?